source/_integrations/fronius.markdown

```diff
@@ -88,6 +98,58 @@ Recommended energy dashboard configuration for meter location in feed in path (`
 - `Battery systems` aren't supported directly. Use [Template](/integrations/template) to split and invert negative values of `Power battery` entity (from power_flow) for charging power (W) and positive values for discharging power (W). Then use [Riemann sum](/integrations/integration/) to integrate each into energy values (kWh).
 - For `Devices` use the Ohmpilots `Energy consumed` entity.
 
+### Consumption path
+
+Recommended energy dashboard configuration for meter location in consumption path (`Meter location: 1`):
+
+This configuration has been possible thanks to [@GianlucaCollot work](https://github.com/GianlucaCollot), honors and credit go to him as I have just reproduced his work and documented it here :)
+
+First you need to create the power entities in your `configuration.yaml`. Make sure to replace the name of the sensor, you may only need to update the IP on the end if it was auto-detected:
+
+```yaml
+template:
+  - sensor:
+    # Positive values are from grid
+    - name: "power_bought"
+      unit_of_measurement: W
+      state: >-
+        {{ max(states('sensor.power_grid_fronius_power_flow_0_XXX_XXX_XXX_XXX') | float, 0) }}
+      device_class: power
+
+    # Negative values are to grid
+    - name: "power_sold"
+      unit_of_measurement: W
+      state: >-
+        {{ max((0 - states('sensor.power_grid_fronius_power_flow_0_XXX_XXX_XXX_XXX') | float), 0) }}
+      device_class: power
+```
+
+then, you need to create the energy entities that get data from the power ones (also in your `configuration.yaml` file):
+
+```yaml
+sensor:
+  - platform: integration
+    source: sensor.power_sold
+    name: energy_sold
+    round: 2
+    
+  - platform: integration
+    source: sensor.power_bought
+    name: energy_bought
+    round: 2
+```
```

Comment:
This is not related to this integration and does not belong in this documentation. Instead you could write a note and link to the integration.

Reply:
These are needed as they are the ones you can select in the Energy Dashboard configuration. No idea why, but it is working that way.

Reply:
Sure, but you can do a lot more with entities. It is not the goal of this page to teach about other integration (like the integration integration).

Thus, this does not belong on this page. Please remove it.

Reply:
I think our focus is different. I'm writting the docs that my future me, as a user, would like to find to configure it's solar installation in HA. This is a 100% needed part for it to work. If I remove this, I'll leave a documentation that doesn't fully allow the user to get it working.

What do you think? I remove this and then... add some text saying that this should be donde in order for it to work? or do I live the average user (not interested in the technical parts) alone?

Reply:
I think you are missing that this document here should *only* be documenting the Fronius integration. 
This does *not* include the `energy` integration which has its own [documentation documents](https://www.home-assistant.io/docs/energy).
And also not Riemann sum and state based template sensors - which also have their own sites. 

It would just add maintenance dept to document all this multiple times.

Reply:
Then I got it wrong as the original document is based on Fronius integration but also related to configure the data you get from this integration in the energy integration. Using it with the energy integration is 1/3 of the document and is what I found missing here. In fact, the matter of the consumption path VS feed in grid is not touch until the energy integration section.

If this page souldn't include information/examples about the energy integration, the Riemann sum or template sensor, this PR for this problem in detail is totally senseless and it's best to close it.